journal/translations/messages.ar.yaml

```diff
@@ -13,4 +13,3 @@ app:
     language_choice: 'اختار اللغة:'
     no_articles: 'لا توجد مقالات متاحة.'
     title: 'عرض Libero'
-direction: rtl
```

Comment:
This was always a temporary measure, replaced with Punic (which contains CLDR data).